chapter5-methods/src/chapter5/methods/Chapter5Methods.java

```diff
@@ -0,0 +1,174 @@
+
+package chapter5.methods;
+
+/* CLASS:   CPT 163  Java Programming I
+*  @author  Sharon R. Tinkey
+*  DATE:    09/04/15
+*  CHAPTER: 3
+*  PROGRAM: Assignment 2: conditions
+*/
+
+import java.util.Scanner;		// required to use Scanner class
+/*
+This program is a menu...
+*/
+public class Chapter5Methods {
+
+    /**
+     * Main presents the menu.
+     * @param  args  no arguments
+     */
+     public static void main(String[] args) {
+     
+    // Create Scanner object for keyboard input
+    Scanner keyboard = new Scanner(System.in);
+    
+    String menuSelStr;          // menu item selected
+    int menuSelNum;             // menu item selected
+ 
+    do {
+        // display menu
+        System.out.println("\n\nMy Java Class Assignments");
+        System.out.println("=========================\n");
+        System.out.println("1. Assignment 1: Word Sum");
+        System.out.println("2. Assignment 2: Conditions");      
+        System.out.println("3. Assignment 3: Reverse the Word");
+        System.out.println("4. Assignment 4: Methods");
+        System.out.println("5. Assignment 5: TBD");
+        System.out.println("6. EXIT\n");
+        System.out.print("Select a program to run: ");
+        menuSelStr = keyboard.nextLine();              // get menu item selected
+        menuSelNum = Integer.parseInt(menuSelStr);    // change item into number
+        
+        System.out.println();                 // output a blank line for spacing
+    
+        // execute the user's selection
+        switch (menuSelStr)  {
```

Comment:
I agree that a switch statement is great for menus I like to see you using it :+1: